components/service/glean/docs/metrics/adding-new-metrics.md

```diff
@@ -80,15 +80,28 @@ Views.loginOpened...
 There are different metrics to choose from, depending on what you want to achieve:
 
 * [Events](#Events): Records events e.g. individual occurences of user actions, say every time a view was open and from where.
-* [Booleans](#Booleans): Records a single truth value, for example "is a11y enabled?"
+* [Booleans](#Booleans): Records a single truth value, for example "is a11y enabled?".
+* [Labeled booleans](#Labeled-booleans): Records multiple truth values, for example which URI schemes were used or not - `"http"`, `"https"`, `"about"`.
 * [Strings](#Strings): Records a single Unicode string value, for example the name of the OS.
-* [String Lists](#String-lists): Records a list of Unicode string values, for example the list of enabled search engines.
+* [Labeled strings](#Labeled-strings): Records multiple Unicode string values. **TODO: what's the use-case here? Should we block these for now?**
+* [String Lists](#String-lists): Records a list of Unicode string values, for example the list of enabled search engines **TODO - i can't think of a good example here**.
```

Comment:
"What is the list of names for the accessibility services available on the device?"

Reply:
I wonder if that should really be a string set then.
I'll file a bug on that.

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1557353